src/main/java/frc/robot/subsystems/funnel/Funnel.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.funnel;
+
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Ports;
+
+/**
+ * this subsystem transfers balls from intake to conveyor
+ */
+public class Funnel extends SubsystemBase {
+    private Victor motor = new Victor(Ports.Funnel.MOTOR);
+
+    public Funnel() {
+        motor.setInverted(Ports.Funnel.IS_INVERTED);
+    }
+
+    /**
+     * this function sets the motor's velocity
+     * @param velocity - the target velocity (m/s)
+     */
+    public void setVelocity(double velocity) {
+        motor.set(velocity);

```

Comment:
First of all this isn't the syntax for set in our motors, and second you should use percent output and not velocity because you don't use PID nor have any sensor